packages/devextreme/scss/widgets/material/gridBase/_index.scss

```diff
@@ -282,14 +283,6 @@ $grid-masterdetail-padding: 16px;
               width: auto;
             }
           }
-
```

Comment:
We tested it together with @EugeniyKiyashko and decided to roll back these changes.
Because in the dxDataGrid we've our button implementation, and without these styles, disabled buttons render incorrectly.